File: php/QueryExample.php
<?php

require __DIR__ . '/../vendor/autoload.php';

use InfluxDB2\Client;
use InfluxDB2\Model\WritePrecision;

$client = new Client([
    "url" => "https://us-west-2-1.aws.cloud2.influxdata.com",
    "token" => "my-token",
    "org" => "my-org",
    "precision" => WritePrecision::S
]);

$queryApi = $client->createQueryApi();
$result = $queryApi->query('from(bucket: "my-bucket") |> range(start: -1d)');

foreach ($result as $table) {
    foreach ($table->records as $record) {
        $measurement = $record->getMeasurement();
        $field = $record->getField();
        $time = $record->getTime();
        $value = $record->getValue();
        print "$time $measurement: $field=$value\n";
    }
}

$client->close();

File: python/QueryExample.py
from influxdb_client import InfluxDBClient

url = 'https://us-west-2-1.aws.cloud2.influxdata.com'
token = 'my-token'
org = 'my-org'
bucket = 'my-bucket'

with InfluxDBClient(url=url, token=token, org=org) as client:
    query_api = client.query_api()

    tables = query_api.query('from(bucket: "my-bucket") |> range(start: -1d)')

    for table in tables:
        for record in table.records:
            print(str(record["_time"]) + " - " + record.get_measurement()
                  + " " + record.get_field() + "=" + str(record.get_value()))
